### PR TITLE
[#146] add outline on all clickable elements

### PR DESCRIPTION
**Pull request by MathildeDuboille**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation content changes

## Other information

Add outline on all clickable elements for keyboard navigation

Issue Number: 146

Closes : https://github.com/theodo/falco/issues/146

### CODE REVIEW

File: frontend/src/components/Root/components/Menu/components/ProjectSettingsMenuContent/ProjectSettingsMenuContent.tsx [hunk 1]
```diff
@@ -43,13 +43,11 @@ export const ProjectSettingsMenuContent: React.FunctionComponent<Props> = ({
   }
 
 
-  const redirectToProjectPage = () => {
-    history.replace(routeDefinitions.projectDetails.path.replace(':projectId', project.uuid))
```
Comment (fargito) [inline frontend/src/components/Root/components/Menu/components/ProjectSettingsMenuContent/ProjectSettingsMenuContent.tsx]: I think you can also remove the `history` import then

File: frontend/src/pages/PagesAndScriptsSettings/PagesAndScriptsSettings.style.tsx [hunk 2]
```diff
@@ -2,13 +2,19 @@ import styled from 'styled-components';
 import { colorUsage, fontFamily, fontSize, fontWeight, getSpacing, lineHeight, settingsContainerSize } from 'stylesheet';
 
 const Style = {
```
Comment (fargito) [inline frontend/src/pages/PagesAndScriptsSettings/PagesAndScriptsSettings.style.tsx]: Could you break the `Style` default export into several exports ?

**Comment by fargito** [comment]
Thanks for this PR !

However I think there is a slight improvement  when I have several projects, when navigating using tab on the "My Projects" menu and toggling it, the next press on tab focuses "My Account" and not the next project.

Do you think you fix this ?

File: frontend/src/components/Root/components/Header/components/AccountMenu/AccountMenu.style.ts [hunk 3]
```diff
@@ -1,5 +1,14 @@
+import Button from "components/Button";
```
Comment (donroyco) [inline frontend/src/components/Root/components/Header/components/AccountMenu/AccountMenu.style.ts:1]: Use single quotes like below.

File: frontend/src/components/MessagePill/MessagePill.style.ts [hunk 4]
```diff
@@ -1,5 +1,6 @@
+import Button from 'components/Button';
 import styled from 'styled-components';
-import { colorUsage, fontSize, getSpacing } from 'stylesheet';
+import {colorUsage, fontSize, getSpacing, inheritVar} from 'stylesheet';
```
Comment (donroyco) [inline frontend/src/components/MessagePill/MessagePill.style.ts:3]: I think this should be:
```
import { colorUsage, fontSize, getSpacing, inheritVar } from 'stylesheet';
```
with spaces before and after.

**Pull request merged**